spyder/api/shellconnect/mixins.py

```diff
@@ -92,89 +201,133 @@ class ShellConnectPluginMixin(ShellConnectMixin):
     # ---- Connection to the IPython console
     # -------------------------------------------------------------------------
     @on_plugin_available(plugin=Plugins.IPythonConsole)
-    def on_ipython_console_available(self):
-        """Connect to the IPython console."""
+    def on_ipython_console_available(self) -> None:
+        """
+        Connect this object to the signals of the IPython Console.
```

Comment:
```suggestion
        Connect this object to the relevant signals of the IPython Console.
```

This doesn't connect to all signals (which is the implication with the current wording).